core/web/migrate_controller.go

```diff
@@ -58,11 +58,29 @@ func (mc *MigrateController) Migrate(c *gin.Context) {
 		jsonAPIError(c, http.StatusInternalServerError, err)
 		return
 	}
+
+	var eiwhSpec *job.ExternalInitiatorWebhookSpec
+	if jbV2.WebhookSpec != nil && len(jbV2.WebhookSpec.ExternalInitiatorWebhookSpecs) > 0 {
+		eiwhSpec = &jbV2.WebhookSpec.ExternalInitiatorWebhookSpecs[0]
+		jbV2.WebhookSpec.ExternalInitiatorWebhookSpecs = make([]job.ExternalInitiatorWebhookSpec, 0)
+	}
+
 	jb, err := mc.App.AddJobV2(c, jbV2, jbV2.Name)
 	if err != nil {
 		jsonAPIError(c, http.StatusInternalServerError, err)
 		return
 	}
+
+	if eiwhSpec != nil {
+		err := mc.App.GetStore().DB.Exec(
+			`INSERT INTO external_initiator_webhook_specs (external_initiator_id, webhook_spec_id, spec) VALUES (?,?,?)`,
```

Comment:
What happens if they specified an external initiator spec on their job? Can we copy that over too?

(I believe the place this used to live is `initiators.body`)

Reply:
yup, i'm copying that now to `external_initiator_webhook_specs` @samsondav